src/test/kotlin/domains/ScoreCalculatorTest.kt

```diff
@@ -0,0 +1,79 @@
+package domains
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+internal class ScoreCalculatorTest {
+
+    @Test
+    @DisplayName("ACE를 갖고있지 않다면 모든 수를 더해준다")
+    fun `sut add all numbers if don't have ACE`() {
+        // Arrange
+        val defeatedMaxNumber = 21
+        val cards = Cards()
+        cards.addCard(Card(PokerNumber.TEN, PokerShape.CLOVER))
+        cards.addCard(Card(PokerNumber.TEN, PokerShape.DIAMOND))
+        cards.addCard(Card(PokerNumber.TEN, PokerShape.HEART))
+        val sut = ScoreCalculator(cards)
+
+        // Act
+        val actual = sut.sumOfNumbers(defeatedMaxNumber)
+
+        // Assert
+        assertThat(actual).isEqualTo(30)
+    }
+
+    @Test
+    @DisplayName("ACE를 갖고있다면 defeatedMaxNumber를 넘기지 않는 선에서 ACE를 11로 취급해서 모든 수를 더해준다")
+    fun `sut add all numbers if have ACE then plus 10`() {
+        // Arrange
+        val defeatedMaxNumber = 21
+        val cards = Cards()
+        cards.addCard(Card(PokerNumber.ACE, PokerShape.CLOVER))
+        cards.addCard(Card(PokerNumber.TWO, PokerShape.CLOVER))
+        val sut = ScoreCalculator(cards)
+
+        // Act
+        val actual = sut.sumOfNumbers(defeatedMaxNumber)
+
+        // Assert
+        assertThat(actual).isEqualTo(13)
+    }
+
+    @Test
+    @DisplayName("ACE를 갖고 있을때의 경우의 수 - 1")
```

Comment:
```suggestion
    @DisplayName("10클로버와 10다이아몬드를 가진 상태에서_ACE다이아몬드를 받으면_21점이다")
```
경우의 수는 어떤 것을 의미할까요?
6개월이 지나고 이 코드를 보았을 때, 어떤 것을 테스트하는지 쉽게 알 수 있을까요?
테스트하고자 하는 내용을 자세하게 적어보면 어떨까요? (이하 관련 내용 동일)

Reply:
그렇군요! 
디스플레이 네임만 읽고도 내용을 알 수 있어야 하는데 제가 작성한 방법은 코드까지 읽어야 했네요!
감사합니다!